### DISCUSSION TITLE
pos invantry

### DISCUSSION THREAD

**Discussion by janithpc** [community/community#141191] [Programming Help]
### Body

i need to crate using php and mysql to make invantry system

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rrRRohith** [comment]
If you're looking to build an inventory system in PHP, I highly recommend using Laravel due to its robust framework and support for database management, security, and user authentication, all of which are essential for an inventory system. Alternatively, you could look into open-source Laravel projects that already have inventory management functionalities. Here are a few solid options:

https://github.com/bagisto/bagisto
https://github.com/uilibrary/stocky
https://github.com/stevebauman/inventory
